spec/services/search_indexer_spec.rb

```diff
@@ -13,14 +13,16 @@
     expect(raw_data.split(' ').length).to eq(2)
   end
 
+  let(:version_num) { 12345 }
   it 'correctly indexes a post' do
     data = "<a>This</a> is a test"
-
+    stub_constant(Search, 'INDEX_VERSION', version_num)
```

Comment:
mocking and stubbing makes me super uneasy. in this context. 

I would ... index something, then update the row to -1 and make sure it gets re-indexed, something like that ... no need for fancy mocking.